widgetastic_patternfly.py

```diff
@@ -392,7 +392,7 @@ def select(self, *levels, **kwargs):
             passed_levels.append(level)
             finished = passed_levels == levels
             link = self.browser.element(
-                self.DIV_LINKS_MATCHING.format(quote(level)), parent=current_div)
+                self.DIV_LINKS_MATCHING.format(txt=quote(level)), parent=current_div)
```

Comment:
For this use case it is perfectly fine. I think we can merge this and I might eventually split the locators to two for each use case to combat the ambiguity.